.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# vagrant
+.vagrant
```

Comment:
A new line at the end of the file is missing.

Reply:
Fixed